File: WEEK1/DAY3/Prog1/P1.c
#include <stdio.h>
int main(){
    int N;
    printf("Count the number of digits in N\n");
    printf("Enter N: ");
    scanf("%d",&N);
    int cnt=0;
    int n=N;
    while(n>0){
        n=n/10;
        cnt=cnt+1;
    }
    printf("%d has %d digits\n",N,cnt);
}

File: WEEK1/DAY3/Prog1/P1.java
import java.util.Scanner;
public class P1 {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.println("Count the number of digits in N");
        System.out.print("Enter N: ");
        int N=sc.nextInt();
        int n=N;
        int cnt=0;
        while (n>0) {
            cnt+=1;
            n/=10;
        }
        System.out.println(N+" has "+cnt+" digits\n");
        sc.close();
    }
}
